code/game/machinery/bots/ed209bot.dm

```diff
@@ -753,14 +753,14 @@ Auto Patrol: []"},
 		if(perp.dna && perp.dna.mutantrace && perp.dna.mutantrace != "none")
 			threatcount += 2
 
-		if(!perp.wear_id)
+		if(!perp.get_visible_id())
 			if(idcheck)
 				threatcount += 4
 			else
 				threatcount += 2
 
 		//Agent cards lower threatlevel.
-		if(perp.wear_id && istype(perp.wear_id.GetID(), /obj/item/weapon/card/id/syndicate))
+		if(perp.get_visible_id() && istype(perp.get_visible_id(), /obj/item/weapon/card/id/syndicate))
```

Comment:
Actually you don't even need anything before the `&&` here because `istype()` doesn't runtime if the thing it's provided with is null